repos/system_upgrade/el7toel8/actors/migratesendmail/actor.py

```diff
@@ -21,14 +23,20 @@ def process(self):
         if not decision or not decision.migrate_files:
             return
 
+        migrated = []
         for f in decision.migrate_files:
-            migratesendmail.migrate_file(f)
+            if os.path.exists(f):
+                migratesendmail.migrate_file(f)
+                migrated.append(f)
+
         list_separator_fmt = '\n    - '
+        summary = 'Uncompressed IPv6 addresses in {}'.format(list_separator_fmt.join(migrated))
+        if not migrated:
+            summary = 'No files migrated, which might be caused by removal of sendmail package during the upgrade.'
```

Comment:
in this case, it's opposite what the title is saying. So the current report is in case of issue saying
* title: sendmail configuration files migrated
* summary: no files migrated, probably due to ..

which is not ideal. I suggest to create even the list of "skipped" files and have the logic:
* if not skipped, print the migrated report (severity low/info)
* else report sendmail configuration files have not been migrated (severity medium)

Reply:
I agree, the report was confusing. It should be better now.